broker-plugin/plugin/src/main/resources/shared/broker.xml

```diff
@@ -73,9 +73,27 @@ under the License.
 
       <acceptors>
          <acceptor name="artemis">tcp://127.0.0.1:61616?tcpSendBufferSize=1048576;tcpReceiveBufferSize=1048576;protocols=CORE</acceptor>
+         <!-- TODO: Remove this once go client supports SASL EXTERNAL -->
```

Comment:
What purpose does the insecure AMQP port serve?

Reply:
Ok, I see we are using for broker management - temporary step assumed.

Reply:
Yes, this is until https://github.com/Azure/go-amqp/pull/5 is merged (or moved to some other source for this module).